bindings/node/lib/clientEncryption.js

```diff
@@ -167,7 +206,8 @@ module.exports = function(modules) {
 
       const bson = this._bson;
       options = sanitizeDataKeyOptions(bson, options);
-      const context = this._mongoCrypt.makeDataKeyContext(provider, options);
+      const dataKeyBson = bson.serialize(Object.assign({ provider }, options.masterKey));
```

Comment:
Due to changes on this line all the other options like for example `keyAltNames`, besides `options.masterKey`, are not being serialized and therefore not passed to the context.
Was this something intentional?

Reply:
hi  @scriste-sv, thanks for pointing this out! we've opened [NODE-3118](https://jira.mongodb.org/browse/NODE-3118) and will be implementing a fix soon